README.md

```diff
@@ -337,7 +337,18 @@ for information related to developing the code.
 
 If you find `aicsimageio` useful, please cite this repository as:
 
-> AICSImageIO Contributors (2021). AICSImageIO: Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python [Computer software]. GitHub. https://github.com/AllenCellModeling/aicsimageio
+> Eva Maxfield Brown, Dan Toloudis, Jamie Sherman, Madison Swain-Bowden, Talley Lambert, AICSImageIO Contributors (2021). AICSImageIO: Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python [Computer software]. GitHub. https://github.com/AllenCellModeling/aicsimageio
```

Comment:
Should this now use the current year?
```suggestion
> Eva Maxfield Brown, Dan Toloudis, Jamie Sherman, Madison Swain-Bowden, Talley Lambert, AICSImageIO Contributors (2022). AICSImageIO: Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python [Computer software]. GitHub. https://github.com/AllenCellModeling/aicsimageio
```

Reply:
Not really. I sort of think of this as "AICSImageIO v4" when we move to [bioio](github.com/bioio-devs/bioio) and release a new thing we will update the citation to 2022 / 2023 (whenever we have our v1 there

Reply:
Got it!

Reply:
I don't quite follow the reasoning.  Shouldn't it just be the year of the most recent release?

Reply:
I treat it as if it was a paper. "When did we write the 'original' paper?" In which case, v4 of aicsimageio was written in 2021 (i think?). You wouldn't update the publish date of a paper.

granted I know software and papers are different but i also think this will help with aggregating the citations together instead of having a bunch of different aicsimageio yearly versions.